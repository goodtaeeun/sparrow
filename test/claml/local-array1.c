int main() {
  int two_dim[1][2] = {1, 2};

  sparrow_print(two_dim[0]);
  sparrow_print(two_dim[0][0]);
}
