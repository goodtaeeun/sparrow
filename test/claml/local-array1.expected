/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "local-array1.c"
int main(void) {
  int two_dim[1][2];

  {
    {
#line 2
      two_dim[0][0] = 1;
#line 2
      two_dim[0][1] = 2;
#line 4
      sparrow_print((int *)two_dim[0]);
#line 5
      sparrow_print(two_dim[0][0]);
    }
    return (0);
  }
}
